games-strategy/augustus/augustus-3.0.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake readme.gentoo-r1
+
+DESCRIPTION="Feature-enhanced open source re-implementation of Caesar III"
+HOMEPAGE="https://github.com/Keriew/augustus"
+SRC_URI="https://github.com/Keriew/augustus/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="uncapped"
+
+DEPEND="
+	dev-libs/expat
+	media-libs/libpng:0=
+	media-libs/libsdl2[joystick,video,sound]
+	media-libs/sdl2-mixer
+"
+RDEPEND="
+	${DEPEND}
+"
+
+src_prepare() {
+	use uncapped && eapply "${FILESDIR}"/${PN}-3.0.1-uncapped.patch
+	cmake_src_prepare
+}
+
+src_install() {
+	cmake_src_install
+	dodir /usr/libexec
+	mv "${ED}/usr/bin/augustus"  "${ED}/usr/libexec/augustus-game" ||
+		die "Failed to rename executable (required to set default resources location)."
+	newbin - augustus-game <<-EOF
```

Comment:
Noticed one last problem while doing final tests, the .desktop is still calling `augustus` rather than `augustus-game`. Fix however you see prefer (sed or patch in prepare works for me).

Other than this, everything seems ready.